src/Common/src/TypeSystem/Ecma/EcmaField.cs

```diff
@@ -278,5 +303,39 @@ public static byte[] GetFieldRvaData(this EcmaField field)
 
             return result;
         }
+
+        /// <summary>
+        /// Retrieves the corresponding static preinitialized data field by looking at various attributes
+        /// </summary>
+        public static FieldDesc GetPreInitDataField(this EcmaField field)
+        {
+            Debug.Assert(field.IsStatic);
+
+            // Early return if there is no custom attributes
+            if (field.CustomAttributes.Count == 0)
```

Comment:
Not sure but it looks like `GetCustomAttributeHandle` would do very little if there are no attributes. Removed.